Python/clinic/bltinmodule.c.h

```diff
@@ -77,7 +77,9 @@ PyDoc_STRVAR(builtin_format__doc__,
 "\n"
 "Return value.__format__(format_spec)\n"
 "\n"
-"format_spec defaults to the empty string");
+"format_spec defaults to the empty string.\n"
```

Comment:
`Python/clinic/bltinmodule.c.h` is generated file. Changes should be made in `Python/bltinmodule.c`. Run `make clinic` for regenerating `Python/clinic/bltinmodule.c.h`.